src/modules/shortcut_guide/shortcut_guide.h

```diff
@@ -32,8 +32,8 @@ class OverlayWindow : public PowertoyModuleIface {
   virtual void destroy() override;
 
 private:
-  TargetState* target_state;
-  D2DOverlayWindow *winkey_popup;
+  std::unique_ptr<TargetState> target_state;

```

Comment:
If it's not too much trouble, can you move this change, and any other change that is not strictly needed for the json library change, to a separate PR?

Reply:
@enricogior I'll create a separate clean up task and move there what's possible.